api/validators/AdminControllerRequests.ts

```diff
@@ -4,8 +4,11 @@ import {
   CreateBonusRequest as ICreateBonusRequest,
   CreateMilestoneRequest as ICreateMilestoneRequest,
   SubmitAttendanceForUsersRequest as ISubmitAttendanceForUsersRequest,
+  ModifyUserAccessLevelRequest as IModifyUserAccessLevelRequest,
   Milestone as IMilestone,
   Bonus as IBonus,
+  UserAccessUpdates, //FIXME: Does this need to be an interface? Nvm, need to validate all types related to it
```

Comment:
I believe the way we structure the requests are that you imported each of the interfaces as Isomething, but implements them in classes within the current file before using them for anything. I believe this is for flexibility and also backward compatibility for in case we need to add new fields inside the classes but we can keep the interface unchanged.